internal/pkg/cli/app_package.go

```diff
@@ -267,7 +313,13 @@ func BuildAppPackageCmd() *cobra.Command {
 			if err != nil {
 				return fmt.Errorf("couldn't connect to application datastore: %w", err)
 			}
-			opts.envStore = store
+			opts.store = store
+
+			sess, err := session.Default()
+			if err != nil {
+				return fmt.Errorf("error retrieving default session: %w", err)
+			}
+			opts.deployer = cloudformation.New(sess)
```

Comment:
is deployer the right term for this? I think this is a project getter. For deploying, you'll want to use a different session, right?

Reply:
Yeah I agree I named it `deployer` because `projectGetter.GetProjectResourcesByRegion` seems to have redundant information and it's under the `deploy` umbrella.
What do you think about `describer`? since under the hood it's doing describe cfn calls

Reply:
I love describer. That's what i was going to call the 'show' family of commands.